mysql-test/suite/clone/include/clone_command.inc

```diff
@@ -0,0 +1,186 @@
+## Clone command test
+
+# These variables can to be set before sourcing this file. Currently we used
+# to test both local and remote clone.
+#
+# 1. Clone command is expected to return error
+# --let clone_err= <error number>
+#
+# 2. Test Remote Clone command. Default is local clone.
+# --let remote_clone = 1
+#
+# 3. Remote Clone command is expected to return error and the error number
+# is different from local clone.
+# --let clone_remote_err = <error number>
```

Comment:
These are not relevant here. I would suggest to omit the numbering "1.", "2.", "3." as well, to avoid the need of any future merge conflicts or adjustments.